### ISSUE TITLE
Responsive grid re-size does not broadcast re-size events

### ISSUE THREAD

**Issue by dsmorse** [dsmorse/gridster.js#16]
When then grid is configured in responsive mode, changes in the browser dimensions result the widgets to have their size recalculated, but it does not broadcast the event, so objects in the widgets do not get notified.  
Manual re-sizing does broadcast events

**Comment by pySilver** [comment]
@dsmorse actually is there a method to repaint layout? I think that breakpoints should be the correct answear for resposivity support. Take a look at http://kenwheeler.github.io/slick/ and how they handle it. 

I think that would be enought to support mobile device – one would just define breakpoinst and correct dimensions.

**Comment by dsmorse** [comment]
yes there is a method which calculates the layout and rebuild the generated stylesheets, this takes care of the internal issues and the widgets get resized correctly..  The problem is for consumers.  When they put content in the widgets, they can get events send to them when the user manually resizes the widget, but if the page size triggers resize or an API call triggers the resize then their is nothing broadcast from grister for the client to consume.  As a consumer of gridster, they should be able to subscribe to resize events and get them all , not just the events triggered by mouse moves

**Comment by zeldich** [comment]
+1
